src/renderer.rs

```diff
@@ -683,3 +695,54 @@ pub fn render_error(
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use pretty_assertions::assert_eq;
+    #[test]
+    fn test_wget_footer() {
+        let to_html = |x| {
+            format!(
+                r#"<div class="downloadDirectory"><p>Download folder:</p><a class="cmd" title="Click to copy!" style="cursor: pointer;" onclick="navigator.clipboard.writeText(&quot;wget -rcnHp -R 'index.html*' {0}/?raw=true'&quot;)">wget -rcnHp -R 'index.html*' {0}/?raw=true'</a></div>"#,
+                x
+            )
+        };
+        let uri = |x| Uri::try_from(x).unwrap();
+
+        let to_be_tested: String = wget_footer(
+            &uri("https://github.com/svenstaro/miniserve/"),
+            Some("Miniserve"),
+            None,
```

Comment:
Why not make all of these test cases into their own unit tests? That way, you can also make it self-documenting by varying the test names.